documentation.md

```diff
@@ -0,0 +1,35 @@
+# Documentation Policies
+
+## Documentation
+
+The project documentation will reside on the `wiki` page or/and in the` doc` folder of the project repository.
+
+When a new feature is added or a significant change is made. Firstly, a design proposal must be created and only once it is accepted and implemented will be added as part of the project documentation.
+
+## Design Proposals
+
+The design proposal is to facilitate communication with team members and be clear about the idea about the implementation. 
+
+A design proposal is necessary only when a new feature is added or a significant change is made. Therefore it is not necessary to do it for simple changes or bug fixes.
+
+To create a design proposal, you must create an `github issue` in the repository of the project and use the template below:
+
+```
+### Title
+A descriptive title for your proposal.
+
+### Overview
+High-level description of your proposal.
+
+### Use Cases
+Use cases for new feature or requirement.
+
+### Design
+Implementation details for your proposal.
+
+### Test Approach
+List planned test approach and tests (e.g., unit, integration).
+
+### Open points
+Open questions concerning your proposal.
```

Comment:
Delete the text "for your proposal", it is redundant.
Open questions concerning your proposal -> Open questions concerning.